app/src/main/java/org/mozilla/fenix/settings/TurnOnSyncFragment.kt

```diff
@@ -22,15 +23,23 @@ class TurnOnSyncFragment : PreferenceFragmentCompat() {
         (activity as AppCompatActivity).supportActionBar?.show()
     }
 
+    override fun onResume() {
+        super.onResume()
+        (activity as AppCompatActivity).title = getString(R.string.preferences_sync)
```

Comment:
We also usually add below this:
``(activity as AppCompatActivity).supportActionBar?.show()``